ext/mbstring/mbstring.c

```diff
@@ -3990,29 +3993,17 @@ static inline zend_long php_mb_ord(const char *str, size_t str_len, zend_string
 		return -2;
 	}
 
-	{
-		mbfl_wchar_device dev;
-		mbfl_convert_filter *filter;
-		zend_long cp;
-
-		mbfl_wchar_device_init(&dev);
-		filter = mbfl_convert_filter_new(enc, &mbfl_encoding_wchar, mbfl_wchar_device_output, 0, &dev);
-		/* If this assertion fails this means some memory allocation failure which is a bug */
-		ZEND_ASSERT(filter != NULL);
-
-		mbfl_convert_filter_feed_string(filter, (unsigned char*)str, str_len);
-		mbfl_convert_filter_flush(filter);
+	/* Some legacy text encodings have a minimum required wchar buffer size;
+	 * the ones which need the most are SJIS-Mac, UTF-7, and UTF7-IMAP */
+	uint32_t wchar_buf[5];
```

Comment:
I'd suggest adding a macro for the 5, so we have a central place to update if this ever changes.

Reply:
Sounds good, will do.

Reply:
Done, please check.